package.json

```diff
@@ -55,7 +55,8 @@
     "compile": "tsc -p ./",
     "watch": "tsc -watch -p ./",
     "pretest": "npm run compile",
-    "test": "node ./out/test/runTest.js"
+    "test": "node ./out/test/runTest.js",
+    "deploy": "vsce publish --yarn"
```

Comment:
Why yarn?

Reply:
Resolved in 190981b. Developer didn't answer original question, but removed the `--yarn` flag.